addons/addon-base-raas/packages/base-raas-services/lib/environment/service-catalog/environment-config-vars-service.js

```diff
@@ -307,9 +307,7 @@ class EnvironmentConfigVarsService extends Service {
       adminKeyPairName,
     };
 
-    if (enableEgressStore && enableEgressStore.toUpperCase() === 'TRUE') {
-      result.egressStoreIamPolicyDocument = JSON.stringify(egressStoreIamPolicyDocument);
-    }
+    result.egressStoreIamPolicyDocument = JSON.stringify(egressStoreIamPolicyDocument);
```

Comment:
Our workspace cfn template expect a value for this attribute, and will error out if we don't provide it. If the attribute is `{}` then `EgressPolicy` is not added. Incase where `Egress` is turned off, the `egressStoreIamPolicyDocument` will be `{}`